docs/development.md

```diff
@@ -97,8 +97,9 @@ To automatically configure your GitHub App, follow these steps:
 1. Go ahead and click the **Register a GitHub App** button.
 1. Next, you'll get to decide on an app name that isn't already taken. Note: if you see a message "Name is already in use" although no such app exists, it means that a GitHub organization with that name exists and cannot be used as an app name.
 1. After registering your GitHub App, you'll be redirected to install the app on any repositories. At the same time, you can check your local `.env` and notice it will be populated with values GitHub sends us in the course of that redirect.
-1. Install the app on a test repository and try triggering a webhook to activate the bot!
+1. Install the app on a test repository.
 1. Restart the server in your terminal (press <kbd>ctrl</kbd> + <kbd>c</kbd> to stop the server)
+1. Try triggering a webhook to activate the bot!
```

Comment:
Great catch! 

I think it would make sense to move the restart step above the install step, because apps might be acting on installation events, too

```suggestion
1. Restart the server in your terminal (press <kbd>ctrl</kbd> + <kbd>c</kbd> to stop the server)
1. Install the app on a test repository.
1. Try triggering a webhook to activate the bot!
```